utils/default values/songs.js

```diff
@@ -1,90 +1,48 @@
+const songIDs = [

```

Comment:
Was actually planning to do something like this but in the module exporting as json